src/pkg/packager/create.go

```diff
@@ -326,7 +326,9 @@ func (p *Packager) addComponent(index int, component types.ZarfComponent, isSkel
 
 	// If any helm charts are defined, process them.
 	for chartIdx, chart := range component.Charts {
-		re := regexp.MustCompile(`\.git$`)
+		// https://regex101.com/r/jYLoUy/1
+		re := regexp.MustCompile(`\.git@`)
```

Comment:
Should the `@` be an optional? How were we getting away with not having it before?

Reply:
this was resolved in https://github.com/defenseunicorns/zarf/pull/1892